weave

```diff
@@ -1713,9 +1713,11 @@ launch_plugin() {
 }
 
 remove_plugin_network() {
-    docker run --rm --net=host \
-        -v /var/run/docker.sock:/var/run/docker.sock \
-        $PLUGIN_IMAGE --remove-network
+    if [ -z $WEAVE_NO_PLUGIN ] && check_docker_server_api_version 1.21 ; then
+        docker run --rm --net=host \
+               -v /var/run/docker.sock:/var/run/docker.sock \
+               $PLUGIN_IMAGE --remove-network
+    fi
```

Comment:
My thinking was that if the user has explicitly called `weave launch-plugin` then we should attempt to do what they asked for; all the other sites happen whether the user asked or not.